### PR TITLE
Feature state clean

### PR DESCRIPTION
**Pull request by planes**
This branch removes "state" from the mission and cleans up the hierarchies. However, it does slightly modify the flow. So it iterates slightly differently, I found that the 1e-10 change in mission convergence slightly changes the sizing loop results.

# Pull Request Checklist
 * Merge in the latest Develop branch changes to your branch
 * Remove .pyc files from your repository:
   * Linux/Mac
     * find . -name *.pyc -delete
   * Windows
     * del /S *.pyc
 * Run automatic regression and makes sure everything is passing
 * Did you make a new regression test that covers your new code?
 * Did you update your docstrings?
 * Did you update your headers to include your name and date?
 * Do a final compare with the Develop branch to be sure what you're changing

### CODE REVIEW

File: regression/scripts/B737/mission_B737.py [hunk 1]
```diff
@@ -84,7 +84,7 @@ def main():
     old_results = load_results()   
 
     # plt the old results
-    plot_mission(results)
+    plot_mission_new(results)
```
Comment (WallyMaier) [inline regression/scripts/B737/mission_B737.py]: what is the purpose of this?

Comment (timdmacdo) [reply regression/scripts/B737/mission_B737.py]: Are we planning to keep both of these around?

Comment (planes) [reply regression/scripts/B737/mission_B737.py]: I fixed this. Since I aliased conditions back it all works.

**Comment by WallyMaier** [comment]
Lift Forward Propulsor needs to be changed as well

**Pull request merged**